platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.roster;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * A utility class to help use Rooster and RosterEntry instances.
+ */
+public final class RosterUtils {
+    private RosterUtils() {}
+
+    /**
+     * Formats a "node name" for a given node id, e.g. "node0" for nodeId == 0.
+     * This name can be used for logging purposes, or to support code that
+     * uses strings to identify nodes.
+     *
+     * @param nodeId a node id
+     * @return a "node name"
+     */
+    @NonNull
+    public static String formatNodeName(final long nodeId) {
+        return "node" + nodeId;
```

Comment:
should this be nodeId +1?  Until the off-by-one on alias name is fixed?

Reply:
Done.